unit_tests/wind_energy/test_abl_src.cpp

```diff
@@ -303,78 +299,50 @@ TEST_F(ABLMeshTest, hurricane_forcing)
 {
     constexpr amrex::Real tol = 1.0e-12;
     populate_parameters();
-    {
-        amrex::ParmParse pp("CoriolisForcing");
-        pp.add("latitude", 90.0);
-    }
-    {
-        amrex::ParmParse pp("HurricaneTempForcing");
-        pp.add("radial_decay", 0.001);
-    }
+
```

Comment:
somehow, it looks like your changes overwrote part of this hurricane forcing test. My guess is it's just a flaw from one of the update/merges with main. We can sort that out pretty easily, I think.

Reply:
Can you help me with this thanks?

Reply:
I think you should be able to do this just from the command line with git. I'll put those commands here. These instructions are assuming that you didn't intend to change anything in this file. If you don't have the exawind repository set as upstream, you'll want to do 
`git remote add upstream https://github.com/Exawind/amr-wind.git`, then
`git fetch upstream`
Then you can do
`git fetch upstream/main unit_tests/wind_energy/test_abl_src.cpp`
At this point, the changes in this file should be reset, so you can go ahead and commit and push them. Listing the commands here to make it as straightforward as possible
`git add unit_tests/wind_energy/test_abl_src.cpp`
`git commit -m "resetting abl src unit test"`
`git push`

Reply:
The second `git fetch ...` command should instead be 
`git checkout upstream/main unit_tests/wind_energy/test_abl_src.cpp`